packages/server-wallet/src/wallet/index.ts

```diff
@@ -131,14 +132,14 @@ export class Wallet implements WalletInterface {
     return {
       outbox: peers.map(recipient => ({
         method: 'MessageQueued',
-        params: {
-          recipient,
-          sender,
-          data: {
+        params: serializeMessage(
```

Comment:
This call to `serializeMessage` calls `serializeState`, which calculates the `channelId` for each channel.